eisbuk-admin/src/components/atoms/BirthdayMenu/BirthdayMenu.tsx

```diff
@@ -37,6 +38,17 @@ const BirthdayMenu: React.FC<Props> = ({ customers, onClickShowAll }) => {
     setBirthdaysAnchorEl(null);
   };
 
+  const calculateTimeDiff =
+    DateTime.now().plus({ day: 1 }).startOf("day").toMillis() -
+    DateTime.now().toMillis();
+  const [timeDiff, setTimeDiff] = useState(calculateTimeDiff);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setTimeDiff(calculateTimeDiff);
+    }, timeDiff);
+    return clearTimeout();
```

Comment:
`setTimeout` returns an id. It needs to be captured and used in the _function_ returned here. The function should invoke `clearTimeout`: we don't want to call it now - we want to return a function that will clear it when invoked (i.e. when/if the component is unmounted).